### ISSUE TITLE
List format breaks when converting text cell to markdown cell

### ISSUE THREAD

**Issue by JensenJiang** [HappenApps/Quiver#822]
Multi-level list in should have 4 spaces indent(or more? at least 2 spaces is invalid). But when a text cell containing multi-level list is converted to markdown cell, the indent is only 2 spaces, which leads to the view breaks.

**Comment by ylian** [comment]
Yeah, you're absolutely right. It should be 4 spaces. http://commonmark.org/help/tutorial/10-nestedLists.html
